queueing_tool/queues/queue_servers.py

```diff
@@ -54,7 +54,7 @@ def poisson_random_measure(t, rate, rate_max):
     Examples
     --------
     Suppose you wanted to model the arrival process as a Poisson
-    random measure with rate function :math:`r(t) = 2 + \sin( 2\pi t)`.
+    random measure with rate function :math:`r(t) = 2 + \\sin( 2\\pi t)`.
```

Comment:
I'm not sure that these latex docstring changes are correct. Things seem to render just fine on [readthedocs.io](https://queueing-tool.readthedocs.io/en/latest/queues.html#queueing-functions).